pkg/models/commandrequest.go

```diff
@@ -4,7 +4,7 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package device
+package models
 
 import "github.com/edgexfoundry/edgex-go/pkg/models"
```

Comment:
I know it was my suggestion that maybe we put the models under pkg/models vs. just models, however now I'm beginning to think this was incorrect, as all the other public code isn't included underneath pkg.  This is simple enough to change back, but maybe worth discussion via email vs. an issue?

Reply:
#91 opened for this topic